### PR TITLE
Add wapblog's domain

### PR DESCRIPTION
**Pull request by fsodic**
* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

Description of Organization
====
Organization Website: https://www.wapblog.id

I'm Fajar Sodik from WapBlog Developer Team
WapBlog.ID is a platform for create a blog.

Reason for PSL Inclusion
====
WapBlog.ID is a platform blog builder who user can create a blog which our subdomain or their own domain. If they don't have a domain, we provide some excelent domain for them like wbq.me or wblog.id and many other.

It could be very usefull, if browser could correctly manage those subdomains (cookies, highlighting the subdomain, sorting by subdomain and not all the domain ). We want these services to be recognized as a known public suffix and wapblog.id services too.

DNS Verification via dig
=======
```
dig +short TXT _psl.bloger.id
"https://github.com/publicsuffix/list/pull/1111"
```

```
dig +short TXT _psl.fastblog.net
"https://github.com/publicsuffix/list/pull/1111"
```

```
dig +short TXT _psl.idnblogger.com
"https://github.com/publicsuffix/list/pull/1111"
```

```
dig +short TXT _psl.indowapblog.com
"https://github.com/publicsuffix/list/pull/1111"
```

```
dig +short TXT _psl.wblog.id
"https://github.com/publicsuffix/list/pull/1111"
```

```
dig +short TXT _psl.wbq.me
"https://github.com/publicsuffix/list/pull/1111"
```

make test
=========
```
============================================================================
Testsuite summary for libpsl 0.21.1
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
```

### CODE REVIEW

**Comment by fsodic** [comment]
@dnsguru is this valid bro?

**Review by dnsguru — CHANGES_REQUESTED** [review]
thank you for submitting through the appropriate process.  
please revise this to sort them by TLD then by SLD

**Comment by fsodic** [comment]
### ****

> thank you for submitting through the appropriate process.
> please revise this to sort them by TLD then by SLD

is this about com, net, and next .id and .me?

File: public_suffix_list.dat [hunk 1]
```diff
@@ -13264,6 +13264,15 @@ v.ua
 // Submitted by Masayuki Note <masa@blade.wafflecell.com>
 wafflecell.com
 
+// WapBlog.ID : https://www.wapblog.id
+// Submitted by Fajar Sodik <official@wapblog.id>
+idnblogger.com
+indowapblog.com
+fastblog.net
```
Comment (dnsguru) [inline public_suffix_list.dat:13271]: move fastblog.net to end

Comment (fsodic) [reply public_suffix_list.dat]: okay, it's done

**Comment by fsodic** [comment]
oh i know,
.com, .id, .me, .net
order on tld extension

**Comment by fsodic** [comment]
@sleevi & @dnsguru is my PR valid?

**Review by fsodic — COMMENTED** [review]
fastblog.net was moved to end

**Review by dnsguru — APPROVED** [review]
- No conflict with base
- DNS validations all present and accurate - domains verified
- Tests pass
- Appropriate and complete rationale

**APPROVED**

This precedes #1116 requirements on proving renewal terms >2y, but @fsodic do be aware later automation review may remove names with <1year from review left in their term (counted from the date of automation run) in the future.

**Comment by dnsguru** [comment]
@fsodic thank you for revising the order of the change ; this has been reviewed and approved.  

The timing of this submission is before the 2y requirement so it will not be enforced here, but as mentioned in the approval.  There is likely to be culling automation that would use the expiry date of domains as determination, so it is wise to ensure renewing names so that automation won't pull names out.

**Comment by fsodic** [comment]
@dnsguru I made a mistake writing a domain name, where I asked bloger.id for this PR but I included bloghp.id.
Can I change it?

**Pull request merged**